modules/floating-buttons/module.php

```diff
@@ -90,6 +90,8 @@ public function __construct() {
 		add_action( 'wp_ajax_elementor_send_clicks', [ $this, 'handle_click_tracking' ] );
 		add_action( 'wp_ajax_nopriv_elementor_send_clicks', [ $this, 'handle_click_tracking' ] );
 
+		add_action( 'wp_enqueue_scripts', [ $this, 'register_style' ] );
```

Comment:
Just making sure this is definitely the correct hook to use? And there's not an Elementor hook we should rather use?

Reply:
Why does it matter? it's only for registration, not for enqueue.